pkg/egressgateway/manager.go

```diff
@@ -87,6 +87,7 @@ var defaultConfig = Config{
 
 func (def Config) Flags(flags *pflag.FlagSet) {
 	flags.Bool("install-egress-gateway-routes", def.InstallEgressGatewayRoutes, "Install egress gateway IP rules and routes in order to properly steer egress gateway traffic to the correct ENI interface")
+	flags.MarkDeprecated("install-egress-gateway-routes", "This option is deprecated, has no effect, and will be removed in v1.16")
```

Comment:
I remember at some point we said that the "FIB lookup" logic would only get enabled iff this option is set. To ensure that the egress interface would have bpf_host attached. Did you come up with another solution, or do we just want to take the simple approach first?

Reply:
I'm just a bit eager to rip out this code (as otherwise it will stay around for yet another stable release) and start testing the feature with fib lookups always enabled

> To ensure that the egress interface would have bpf_host attached

I thought we concluded that these interfaces will have already bpf_host attached anyway (otherwise it's not possible to reverse translate reply traffic) so that shouldn't be an issue

Reply:
> I thought we concluded that these interfaces will have already bpf_host attached anyway (otherwise it's not possible to reverse translate reply traffic) so that shouldn't be an issue

Ah yes that sounds reasonable (and I probably just don't remember discussing it before :disappointed:). Agree with the sentiment to roll this out ASAP.